### DISCUSSION TITLE
Avoid Countries Feature

### DISCUSSION THREAD

**Discussion by xwing007** [albar965/littlenavmap#979] [Ideas]
Hi there.
I would like to see the Avoid Countries Feature.
Regards
Peter

**Comment by albar965** [comment]
Sorry, but very unlikely to happen. The intersection calculation with country borders (I have none yet in LNM) while finding the next waypoint during calculation is close to undoable. :man_shrugging: 
Alex

**Reply by xwing007** [reply]
No problem, ;-)
You still does great work.
Thanks.
Peter
